### PR TITLE
Alerting: separate namespace & group inputs for system alerts

### PR DESCRIPTION
**Pull request by domasx2**
System alert form new has separate dropdowns for namespace & group.
It's possible to select "+ Add new" option and create new namespace or group.

https://user-images.githubusercontent.com/847684/114864653-ecc9d580-9df9-11eb-8eb7-3bac0641df63.mp4

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/features/alerting/unified/components/rule-editor/SelectWIthAdd.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      <Select
+        width={width}
+        options={_options}
+        value={value}
+        className={className}
+        placeholder={placeholder}
+        disabled={disabled}
+        onChange={(val: SelectableValue) => {
+          const value = val?.value;
+          console.log('select', value);
```
Comment (mckn) [inline public/app/features/alerting/unified/components/rule-editor/SelectWIthAdd.tsx:66]: nit: should be removed prior to merge.

**Review by mckn — COMMENTED** [review]
I think this looks good! Overall a great job.

It would probably be a good idea to introduce good named helper functions to prevent using comments so much but at the same time I guess it is a matter of taste.

File: public/app/features/alerting/unified/components/rule-editor/SelectWIthAdd.tsx (hunk 1 above)
Comment (domasx2) [reply public/app/features/alerting/unified/components/rule-editor/SelectWIthAdd.tsx:66]: - [ ] oops

**Review by mckn — APPROVED** [review]
LGTM

**Pull request merged**